### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by KorDum**
Some code cleanup.
Remove some unused classes and methods.

### CODE REVIEW

**Comment by KorDum** [comment]
Hmm. Travis didn't even get to the build.

File: src/main/java/binnie/core/gui/BinnieCoreGUI.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package binnie.core.gui;
 
-import binnie.core.craftgui.WindowFieldKit;
+import binnie.botany.craftgui.WindowFieldKit;
```
Comment (mezz) [inline src/main/java/binnie/core/gui/BinnieCoreGUI.java]: this import change here broke the travis build

**Review by mezz — CHANGES_REQUESTED** [review]

**Comment by KorDum** [comment]
Fixed, thanks.

File: src/main/java/binnie/core/machines/RendererMachine.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import net.minecraft.world.IBlockAccess;
 import net.minecraft.world.World;
 
-// TODO unused class?
+// Hmm. It's used class
```
Comment (mezz) [inline src/main/java/binnie/core/machines/RendererMachine.java:12]: This is not a very good comment...

File: src/main/java/binnie/core/triggers/TriggerWorking.java [hunk 3]
```diff
@@ -1,43 +0,0 @@
-package binnie.core.triggers;
-
-import binnie.core.machines.Machine;
-import binnie.core.machines.power.IProcess;
-
-// TODO unusing class?
-public class TriggerWorking {
```
Comment (mezz) [inline src/main/java/binnie/core/triggers/TriggerWorking.java:7]: I think these triggers are supposed to be used by BuildCraft logic gates, are you sure they can not work?

**Review by mezz — APPROVED** [review]

File: src/main/java/binnie/core/triggers/TriggerWorking.java (hunk 3 above)
Comment (KorDum) [reply src/main/java/binnie/core/triggers/TriggerWorking.java:7]: Without implementing some BuildCraft API interfaces?
Need to check it, of course. I'll check during the day.

Comment (mezz) [reply src/main/java/binnie/core/triggers/TriggerWorking.java:7]: Yeah I guess it was never complete. I will merge this.

File: src/main/java/binnie/core/machines/RendererMachine.java (hunk 2 above)
Comment (KorDum) [reply src/main/java/binnie/core/machines/RendererMachine.java:12]: IDEA says that class is unused. But if drop it, genetic machines aren't render o_O
Many strange code...

**Pull request merged**